main.go

```diff
@@ -486,10 +486,8 @@ func processLogFile(file string, enc *json.Encoder) (int, error) {
 
 	if offset > 0 {
 		if offset == info.Size() {
-			if plugin.Verbose {
-				fmt.Printf("Cached offset in state directory for %s indicates file not updated since last read\n", file)
-				return 0, nil
-			}
+			fmt.Printf("Cached offset in state directory for %s indicates file not updated since last read\n", file)
```

Comment:
Shouldn't the output be suppressed when verbose is disabled?

Reply:
sure that's reasonable. let me change that real quick

Reply:
updated in latest commit